src/commands/clusters/create.ts

```diff
@@ -111,7 +112,16 @@ export default class ClusterCreate extends BaseCommand {
         type: 'input',
         name: 'cluster',
         message: 'What would you like to name your new cluster?',
-        when: !args.cluster,
+        when: () => {
+          if (args.cluster) {
+            return false;
+          }
+
+          if (isCi) {
```

Comment:
Should this be CI or should we throw on all TTY? Is there more context here @ryan-cahill

Reply:
We might as well add a check for `process.env.isTTY` since it's in the [inquirer docs](https://github.com/SBoudrias/Inquirer.js/#use-in-non-interactive-environments)

Reply:
`process.env.isTTY` is undefined. I ended up just re-using `prompts_available()`.